js/index.js

```diff
@@ -1,7 +1,41 @@
-// Iteration 1: Names and Input
+//it 1 
+var hacker1 = 'Jason';
+console.log(`"The driver's name is ${hacker1}"`);
```

Comment:
If you use the string interpolation, you don't need the quote inside, but you solve that later in the exercises, so good :)